crates/sui-indexer-alt-framework/src/pipeline/concurrent/mod.rs

```diff
@@ -218,6 +221,23 @@ pub(crate) fn pipeline<H: Handler + Send + Sync + 'static>(
     let pruner_cancel = cancel.child_token();
     let handler = Arc::new(handler);
 
+    // The watch channel is an Option<channel<Option<u64>>>. The outer Option indicates whether the
+    // pipeline is tasked or main. The inner is the safety mechanism - consumers must wait until the
+    // task has started and provided a fresh value.
+    let (main_reader_lo_tx, main_reader_lo_rx) = if task.is_some() {
+        let (tx, rx) = watch::channel(None);
+        (Some(tx), Some(rx))
+    } else {
+        (None, None)
+    };
+
+    let main_reader_lo_task = main_reader_lo::<H>(
```

Comment:
minor nit: the name isn't sitting quite right with me -- the other tasks are named for verbs, while this one is named for a noun. If we were to name it for what it was doing, what would that be?